src/WinSW/Program.cs

```diff
@@ -986,10 +961,82 @@ static void Elevate(bool noElevate)
                     Environment.Exit(e.ErrorCode);
                 }
             }
+
+            static void AutoRefresh(XmlServiceConfig config)
+            {
+                if (!config.AutoRefresh)
+                {
+                    return;
```

Comment:
Put a message to the log? Could help to raise awareness about the feature

Reply:
Makes sense. Which log level will be appropriate?

Reply:
INFO?

Reply:
I enabled this feature by default... Any problems?